CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.67.0
```

Comment:
You don't need to mention a number here. Just leave it like `## newVerion`, then I will bump it up.

Reply:
Done